app/src/main/res/values/colors.xml

```diff
@@ -87,7 +87,7 @@
     <color name="contrast_text_dark_theme">@color/primary_text_dark_theme</color>
     <color name="caption_text_dark_theme">@color/photonLightGrey70</color>
     <color name="foundation_dark_theme">#1C1B22</color>
-    <color name="inset_dark_theme">#32313C</color>
+    <color name="inset_dark_theme">@color/photonDarkGrey10</color>
```

Comment:
I think I made an error. We actually need to check all instances of `inset_dark_theme` and cannot simply change this color. We might want to create a custom variable just for the background here avoid changing all the UI components that might use inset_dark_theme, but was not covered by the original UX review.

Reply:
Okay, I'll check the instances of `inset_dark_theme` 👍